Code-the-Dream-School/ffprac-team2-front
# Prac 9 navigation frontend

## One Line Description
Navigation with interactive buttons: 
unregistered users: Home, Browse Tutors, Login.
for Tutor role: Home, My dashboard, Browse Tutors, Tutor-profile, Logout.
The “Tutor Profile” button is a round button with initials (avatar).
for Parent role: Home, My dashboard button, Browse Tutors, Logout.
Burger menu for small screens.

## Requirements
Navigation with interactive buttons: 
Clicking on the Home, reloads the Landing page
Clicking on the Login loads the Login page
Clicking on the Browse Tutor loads the Tutor search page
for tutor role: 
Clicking on the My dashboard redirects to the Tutor dashboard page
Clicking on the Tutor-profile redirects to the tutor-profile page
for parent role:
Clicking on the My dashboard redirects to the Parent-dashboard page
Clicking on the Logout button redirects to the home page (and clear localStorage)

_What this code should do_

## Notes

_Any additional info_

## Test Steps
for unregistered users: 
check that buttons are present Home, Browse Tutors, and Login.
all buttons have to redirect to the proper pages.
for tutor role:
-Create or Login as a tutor
check if buttons are present: Home, My dashboard, Browse Tutors, Tutor-Profile, Logout. All buttons have to redirect to the proper pages.

for parent role:
-Create or Login as a parent
check if buttons are present: Home, My dashboard, Browse Tutors, Logout.
All buttons have to redirect to the proper pages.

_Instructions on how to test the changes_

## UI/UX Outcome
<img width="1440" alt="Screen Shot 2024-04-16 at 2 52 01 PM" src="https://github.com/Code-the-Dream-School/ffprac-team2-front/assets/10125103/04cfa192-14a5-4caf-b6a6-5967ff11c137">
<img width="1440" alt="Screen Shot 2024-04-16 at 2 55 24 PM" src="https://github.com/Code-the-Dream-School/ffprac-team2-front/assets/10125103/fe706145-6a6b-4abb-915e-be4b19ac4980">

<img width="1440" alt="Screen Shot 2024-04-16 at 2 52 15 PM" src="https://github.com/Code-the-Dream-School/ffprac-team2-front/assets/10125103/2dd3e219-0139-4ec6-8bbd-c0a46ac92683">
<img width="455" alt="Screen Shot 2024-04-16 at 2 52 55 PM" src="https://github.com/Code-the-Dream-School/ffprac-team2-front/assets/10125103/023847b5-17fd-41e7-848c-a8697d9f1774">
<img width="446" alt="Screen Shot 2024-04-16 at 2 52 35 PM" src="https://github.com/Code-the-Dream-School/ffprac-team2-front/assets/10125103/09d57d98-1db7-442f-8d53-f64625e1bd35">
<img width="443" alt="Screen Shot 2024-04-16 at 2 53 21 PM" src="https://github.com/Code-the-Dream-School/ffprac-team2-front/assets/10125103/a0261de7-b799-48ba-8aa2-c0a7396f4c7d">
